persistence/src/main/resources/db/migration/data/integration/V1_12_3__Add_statehistories.sql

```diff
@@ -1,6 +1,6 @@
-INSERT INTO states_history (id,application_id,creation_date,language_skill,description,first_test_result,offered_money,claim,feedback_date,day_of_start,state_id) VALUES (1,1,'2016-07-26 11:48:55',NULL,'application description',NULL,NULL,NULL,NULL,NULL,1);
-INSERT INTO states_history (id,application_id,creation_date,language_skill,description,first_test_result,offered_money,claim,feedback_date,day_of_start,state_id) VALUES (2,1,'2016-07-26 11:48:55',NULL,NULL,NULL,NULL,NULL,NULL,NULL,2);
-INSERT INTO states_history (id,application_id,creation_date,language_skill,description,first_test_result,offered_money,claim,feedback_date,day_of_start,state_id) VALUES (3,1,'2016-07-26 11:48:55',8,'hr desc',NULL,NULL,NULL,NULL,NULL,3);
-INSERT INTO states_history (id,application_id,creation_date,language_skill,description,first_test_result,offered_money,claim,feedback_date,day_of_start,state_id) VALUES (4,1,'2016-07-26 11:48:56',NULL,NULL,90,NULL,NULL,NULL,NULL,4);
-INSERT INTO states_history (id,application_id,creation_date,language_skill,description,first_test_result,offered_money,claim,feedback_date,day_of_start,state_id) VALUES (5,2,'2016-07-26 11:48:57',NULL,NULL,90,NULL,NULL,NULL,NULL,4);
-INSERT INTO states_history (id,application_id,creation_date,language_skill,description,first_test_result,offered_money,claim,feedback_date,day_of_start,state_id) VALUES (6,3,'2016-07-26 11:48:58',NULL,NULL,90,NULL,NULL,NULL,NULL,4);
+INSERT INTO states_history (id,application_id,creation_date,description,first_test_result,offered_money,claim,feedback_date,day_of_start,state_id) VALUES (1,1,'2016-07-26 11:48:55','application description',NULL,NULL,NULL,NULL,NULL,1);
```

Comment:
While changing an existing SQL migration for the integration profile is safe, it is considered a bad habit.

For a living database introducing this change would require extra work, so what if you already write the SQL as it were a living database and leave flyway to migrate it into the actual database when needed.